development/prs_and_reviewing.ipynb

```diff
@@ -226,7 +226,7 @@
    "name": "python",
```

Comment:
This PR should have no changes to the notebooks. One way to remove it is to add a new commit that revert this change. An even better one is to make that revert commit and then with interactive rebase squash it into your first commit.

But, overall a revert is good enough:

Assuming you have `bsipocz` as a name for the class repo, and `PCostaUW` as the name for your fork. Update the command if that's not the case:

```
# check out the branch of this PR:
git checkout personal_assignments

# check out the version of the notebook from my `main` branch. First argument is the reference to the version, second is the path to the file
git checkout bsipocz/main development/prs_and_reviewing.ipynb

# check status, the file should come up as `modified` with the reverted changes
git status

# check the diff, as the file is already in the staged area, use HEAD
git diff HEAD

# for the diff you should see something like this:
```
```
$ ~/munka/teaching/ASTR497 [:3004a46|●1…2] $ gitd HEAD
diff --git a/development/prs_and_reviewing.ipynb b/development/prs_and_reviewing.ipynb
index e39edec..b198d77 100644
--- a/development/prs_and_reviewing.ipynb
+++ b/development/prs_and_reviewing.ipynb
@@ -226,7 +226,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.5"
+   "version": "3.9.1"
   },
   "toc": {
    "base_numbering": 1,
```
    
```
# add commit
git commit -m "reverting changes to notebook"
    
# push back to the branch on your fork, the PR will be updated automatically
git push PCostaUW personal_assignments
    
```